static/style.css

```diff
@@ -1,17 +1,78 @@
-.main {
-    display: flex;
-    min-height: 100vh;
-    flex-direction: column;
+body {
+	display: flex;
+	height: 100vh;
+	width: 90vw;
+	margin: auto;
+	flex-direction: column;
+	justify-content: center;
+	align-self: center;
 }
 
-.touchpad {
-    background-color: black;
-    min-height: 28vh;
-    display: flex;
-    justify-content: center;
-    align-items: center;
+header {
+	height: 10%;
+	width: 100%;
+	text-align: center;
+	background-color: rgb(35, 77, 141);
+	font-family: Arial, Helvetica, sans-serif;
+	color: white;
+}
+
+main {
+	height: 80%;
+}
+
+footer {
+	height: 8%;
+	text-align: center;
+	display: flex;
+}
+footer p {
+	align-self: center;
+	margin: auto;
 }
-  
+
+footer a:hover {
+	color: brown;
+}
+
 .container {
-    flex: 1;
-}
\ No newline at end of file
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-rows: 1fr 1fr 1fr 1fr 4fr;
+}
+
+.play {
```

Comment:
Looks identical to .mute. Can we make them implement a common class?

Reply:
I think so